homeassistant/helpers/event.py

```diff
@@ -70,6 +72,7 @@ def track_point_in_utc_time(hass, action, point_in_time):
     point_in_time = dt_util.as_utc(point_in_time)
 
     @ft.wraps(action)
+    @asyncio.coroutine
```

Comment:
Unless this usually returns early before fire_action, it might not help alot to keep it on the loop. If it does though.... definitely better to avoid jamming up the thread-pool.